packages/devtools_app/lib/src/network/flutter/http_timing_tab.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../http_request_data.dart';
+
+/// A [Widget] which displays timing information for an HTTP request.
+class HttpRequestTimingTab extends StatelessWidget {
+  const HttpRequestTimingTab(this.data);
+
+  @override
+  Widget build(BuildContext context) {
+    final events = <Widget>[];
+    for (final instant in data.instantEvents) {
+      final timeDiffMillis = instant.timeDiffMs;
+      events.add(_buildTile(
+          instant.name, [_buildRow('Duration', '$timeDiffMillis ms')]));
```

Comment:
I think there are some existing helper methods to display times as strings in the devtools code base.